src/Main.jl

```diff
@@ -10,33 +10,34 @@ module MainLoop
 	mutable struct Main
 		assets::String
 		autoScaleZoom::Bool
-		cameraBackgroundColor
+		cameraBackgroundColor::Tuple{Int64, Int64, Int64}
 		close::Bool
 		currentTestTime::Float64
-		debugTextBoxes
-		events
-		globals
-		input
+		debugTextBoxes::Vector{UI.TextBoxModule.TextBox}
+		events::Vector # what is this for ?
+		globals::Vector  # what is this for ?

```

Comment:
This `globals` is a workaround I've created for game projects to use external packages without having to add them to JulGame so I can compile them

Reply:
Cool!

Reply:
Could we call this `Vector{Any}` then?

Reply:
Yeah that'll work!